llvm/lib/Target/AMDGPU/SIInstrInfo.td

```diff
@@ -1710,6 +1710,8 @@ def VOP3PMadMixBF16Mods : ComplexPattern<untyped, 2, "SelectVOP3PMadMixBF16Mods"
 def VINTERPMods  : ComplexPattern<untyped, 2, "SelectVINTERPMods">;
 def VINTERPModsHi  : ComplexPattern<untyped, 2, "SelectVINTERPModsHi">;
 
+def Ignore : ComplexPattern<untyped, 0, "SelectIgnore">;
```

Comment:
@s-barannikov @jurahul @Pierre-vh FYI - this patch introduces this dummy complex pattern as a way to let you ignore an operand in an isel pattern. I wonder if tablegen should have some kind of built-in support for this instead?